samples/CommunityToolkit.Maui.Sample/MauiProgram.cs

```diff
@@ -216,4 +215,15 @@ static void RegisterEssentials(in IServiceCollection services)
 	{
 		return services.AddTransientWithShellRoute<TPage, TViewModel>(AppShell.GetPageRoute<TViewModel>());
 	}
+
+	class MobileHttpRetryStrategyOptions : HttpRetryStrategyOptions
```

Comment:
I believe we can make this one `sealed` since it will not be extended
```suggestion
	sealed class MobileHttpRetryStrategyOptions : HttpRetryStrategyOptions
```

Reply:
Absolutely! Good catch!